lib/user.js

```diff
@@ -475,8 +475,8 @@ var User = module.exports = function(ncmb){
 
     return ncmb.request({
       path: "/" + ncmb.version + "/login",
-      method: "GET",
-      query: {
+      form: "POST",
```

Comment:
こちらの修正はformではなくmethodだと思います
```suggestion
      method: "POST",
```

Reply:
修正しました